module-apps/application-call/ApplicationCall.cpp

```diff
@@ -181,10 +184,7 @@ namespace app
             return sys::MessageNone{};
         });
 
-        connect(typeid(AudioEventRequest), [&](sys::Message *request) {
-            auto message = static_cast<AudioEventRequest *>(request);
-            return handleAudioMessageEvent(message);
-        });
+        connect(typeid(AudioEventRequest), [&](sys::Message *request) { return handleAudioMessageEvent(); });
```

Comment:
```suggestion
        connect(typeid(AudioEventRequest), [this](sys::Message *request) { return handleAudioMessageEvent(); });
```
Try to capture only what you need/use.

Reply:
fixed